src/platforms/px4_log.h

```diff
@@ -115,7 +115,7 @@ static inline void do_nothing(int level, ...)
 
 #else
 
-#define __STDC_FORMAT_MACROS
+#define __STDC_FORMAT_MACROS 1
```

Comment:
We might want to delete the 1 here, it wasn't used elsewhere in the code. Or we should make the other occurrences similar, not a big deal.